src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/ClassifiedTextElementComparer.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
```

Comment:
❗ Needs to be `IEqualityComparer<ClassifiedTextElement?>`. Ideally `GetHashCode` should support a null argument.